applications/stepswitch/src/stepswitch_util.erl

```diff
@@ -300,10 +299,11 @@ maybe_endpoint_format_from(Endpoint, Number, DefaultRealm) ->
 
 -spec endpoint_format_from(wh_json:object(), ne_binary(), api_binary(), wh_json:object()) ->
                                   wh_json:object().
-endpoint_format_from(Endpoint, Number, DefaultRealm, CCVs) ->
-    case wh_json:get_value(<<"From-URI-Realm">>, CCVs, DefaultRealm) of
+endpoint_format_from(Endpoint, Number, OffnetReq, CCVs) ->
```

Comment:
the change of `DefaultRealm` to `OffnetReq` needs a
change in spec from `api_binary()` to `wapi_offnet_resource:req()`